framework/include/utils/MooseTypes.h

```diff
@@ -677,7 +677,9 @@ enum EigenSolveType
   EST_KRYLOVSCHUR,     ///< Krylov-Schur
   EST_JACOBI_DAVIDSON, ///< Jacobi-Davidson
   EST_NONLINEAR_POWER, ///< Nonlinear inverse power
-  EST_NEWTON,          ///< Newton-based eigen solver
+  EST_NEWTON, ///< Newton-based eigensolver with an assembled Jacobian matrix (fully coupled by default)
```

Comment:
What do you mean `fully coupled by default`? You mean the matrix coupling pattern is full?

Reply:
Yes

Reply:
You might never use it in griffin

Reply:
hehe, I am actually going to use it.